molecule/default/molecule.yml

```diff
@@ -21,6 +21,8 @@ provisioner:
         ansible_python_interpreter: '{{ ansible_playbook_python }}'
   env:
     ANSIBLE_FORCE_COLOR: 'true'
+  options:
+    vvv: True
```

Comment:
I usually use default verbosity on test runs (when not specifically debugging an issue), but that's just me. @willthames and @fabianvf especially—do you prefer having more verbosity? I can go either way.

Reply:
Definitely prefer more verbosity - lets me validate far more assumptions (which python is in use, which module is actually being exercised) and shows the full invocation, diff and results in the output.

Reply:
I prefer greater verbosity as well, annoying to have to rerun a failed build by hand when the cause of an error is unclear